package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "babel-core": "^6.9.0",
+    "babel-plugin-transform-amd-system-wrapper": "^0.1.0",
```

Comment:
:+1: Hint: I've removed the remap function in the cjs and amd tranfsformers as they aren't referenced by any code.